zbx_redis_stats.py

```diff
@@ -10,9 +10,9 @@
 parser.add_argument('-a','--auth',dest='redis_pass',action='store',help='Redis server pass',default=None)
 args = parser.parse_args()
 
-zabbix_host = '127.0.0.1'	# Zabbix Server IP
-zabbix_port = 10051			# Zabbix Server Port
-hostname = 'redis.srv.name'	# Name of monitored server like it shows in zabbix web ui display
+zabbix_host = '127.0.0.1'       # Zabbix Server IP
+zabbix_port = 10051             # Zabbix Server Port
+hostname = 'redis.srv.name'     # Name of monitored server like it shows in zabbix web ui display
```

Comment:
The problem is that, we have two kinds of `hostname`,
 - the Redis server hostname (`localhost`, `127.0.0.1` or `redis.srv.name`), i.e., the one you put in argument
 - the hostname configured in Zabbix Web UI, i.e., the one you use `hostname` variable.

If they are the same, then there is no problem; if they are different, the user will be mislead.